src/lib/logger.service.ts

```diff
@@ -181,10 +186,14 @@ export class NGXLogger {
 
       logObject.message = message;
 
+      const options = {
+        headers: this.httpService.setHeaders(this._customHttpHeaders),
+        params: this.httpService.setParams(this._customParams)
```

Comment:
-wanted to keep a request builder { building headers, params, methods } as a different file, have incorporated inline, since its a small change  - done